aiida/cmdline/commands/cmd_process.py

```diff
@@ -185,11 +185,33 @@ def process_call_root(processes):
 @click.option(
     '-m', '--max-depth', 'max_depth', type=int, default=None, help='Limit the number of levels to be printed.'
 )
+@click.option(
+    '-e', '--end', 'last', is_flag=True, default=False, help='Report for the most recent node in verdi process list -a'
+)
 @decorators.with_dbenv()
-def process_report(processes, levelname, indent_size, max_depth):
+def process_report(processes, levelname, indent_size, max_depth, last):
     """Show the log report for one or multiple processes."""
     from aiida.cmdline.utils.common import get_calcjob_report, get_process_function_report, get_workchain_report
-    from aiida.orm import CalcFunctionNode, CalcJobNode, WorkChainNode, WorkFunctionNode
+    from aiida.orm import CalcFunctionNode, CalcJobNode, WorkChainNode, WorkFunctionNode, load_node
+    from aiida.tools.query.calculation import CalculationQueryBuilder
+
+    if last:
+        # query all nodes, getting the last one available
+        builder = CalculationQueryBuilder()
+        lastnode_info = builder.get_projected(builder.get_query_set(), CalculationQueryBuilder.default_projections)[-1]
+        # take pk from node info and load that node
+        pk = lastnode_info[0]
+        node = load_node(pk)
+        # set process list to the singular final node
+        processes = [node]
+
+        echo.echo(f'Reporting for final node (pk: {pk})')
+
+        try:
+            remote_dir = node.outputs.remote_folder.get_attribute('remote_path')
+            echo.echo(f'Remote Directory: {remote_dir}')
+        except AttributeError:
+            echo.echo(f'No Remote Directory Found')
```

Comment:
Why did you add this?

Reply:
It's purely for debugging reasons, before I learned about `verdi calcjob gotocomputer`. This can either be removed or set to `echo_info`.

I still use it occasionally since it lets me `cat {remote_dir}/_aiidasubmit.sh` without leaving the current working dir, for example. But just like the `Reporting for final node` above, it's not strictly _required_ info.

Reply:
> It's purely for debugging reasons, before I learned about `verdi calcjob gotocomputer`. This can either be removed or set to `echo_info`.

I don't like it so much because it is specific to `CalcJob`s and doesn't apply to any other processes like calcfunctions, `WorkChains` etc.

> I still use it occasionally since it lets me `cat {remote_dir}/_aiidasubmit.sh` without leaving the current working dir, for example. 

This will only work if your `CalcJob` ran on the `localhost` though. If you ran it on a remote computer, this won't work and you would anyway have to use `verdi calcjob gotocomputer`.

> But just like the `Reporting for final node` above, it's not strictly _required_ info.

The difference is that this applies to all processes. In addition, it can actually help to prevent confusion. Since when you use this flag you think you might get the last node, but another one could have run in the meantime (if the daemon is running and has other proceses running), and so really you got another node than you were anticipating. In this case an explicit reference of exactly which node was matched can prevent confusion.